src/libs/OptionsListUtils.ts

```diff
@@ -1297,6 +1335,12 @@ function getValidOptions(
         return true;
     });
 
+    let workspaceChats: ReportUtils.OptionData[] = [];
+
+    if (shouldSeparateWorkspaceChat) {
+        workspaceChats = allReportOptions.filter((option) => option.isOwnPolicyExpenseChat);
```

Comment:
```suggestion
        workspaceChats = allReportOptions.filter((option) => option.isOwnPolicyExpenseChat && !option.private_isArchived);
```

We need to filter out archived workspaces.

<img width="381" alt="Screenshot 2024-12-20 at 19 57 11" src="https://github.com/user-attachments/assets/33582e19-5bbe-424c-bb26-e8644f013974" />

Reply:
Updated, thanks!